### PR TITLE
Adding Unsupported File component

### PR DESCRIPTION
**Pull request by JimBratsos**
- Added Unsupported File component

- Changed SCSS values of Visualizer and App because it was too small ( screenshot provided in group chat in ztm group)

Problems:
While component is fully responsive, I have my thoughts/problems;

1. When I make the page really small ( at the point where even the text goes out of buttons ) the message gets really small in the component. It is responsive but barely visible. Plus, two lines of text are hard to fit there anyway.

 2. I did not implement the close button function . I left it empty. The reason is that basically the idea is that the upload fails and that we go back to song upload page. If so, then just closing the component will take us to the visualizer which is not what we want. So I left function empty, for now.

Thanks for the help

### CODE REVIEW

File: src/components/UnsupportedFile/unsupportedFile.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                <CloseBtn
+                className = {classes.close}
+                onClick = {handleBack}
+                />
+            </div>
+        </div>
+    );
+};
+
+export default unsupportedFile;
```
Comment (ArnasDickus) [inline src/components/UnsupportedFile/unsupportedFile.js]: Needs a new line, try running `npm run format`

File: src/components/UnsupportedFile/unsupportedFile.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+import React from 'react';
+
+import { ReactComponent as CloseBtn } from '../../assets/UnsupportedFileAssets/close.svg';
+
+import classes from './unsupportedFile.module.scss';
+
+const unsupportedFile = () =>{
+
+    const handleBack = () =>{};
```
Comment (ArnasDickus) [inline src/components/UnsupportedFile/unsupportedFile.js]: If you not going to use it don't forget to remove this function.

**Review by ArnasDickus — CHANGES_REQUESTED** [review]
Hello, sorry for late review, was busy with not programming related stuff.

Anyways I see you haven't added 
`target="_blank" without rel="noopener noreferrer"`, From your last PR.

1) Component responsiveness seems fine, however you should try to increase text-size, button size, On the mobile view, because it's way too small.
2) Yes, you correct about the exit button. You can leave it as it's or If you want to challenge yourself, I think you should try to implement an unsupported format error on AppBrowser.js instead of App.js and if wrong file added, it should not redirect the user to app.js page. You can try discussing with Yogiha about this since he created this AppBrowser.js file.

**Comment by ArnasDickus** [comment]
Hello @JimBratsos how's it going with the task?

**Comment by JimBratsos** [comment]
Hi @ArnasDickus . I have updated it almost a week ago. Now it has some conflicts to be resolved from what I checked. I cannot be of assist at the moment though, meaning I cannot solve the conflicts due to being extremely busy with university. Someone could fill in and complete it, and if possible, put me as a co-author of the component and use my component as well. I would complete it, I did 1 week ago but I assumed you were busy so I did not tag you to check it out.

**Comment by ArnasDickus** [comment]
@JimBratsos Hello, you should have tagged me, I must have missed it. I will review this again tomorrow and fix the conflicting file for you.

**Comment by ArnasDickus** [comment]
Hello @JimBratsos I fixed your merging conflicts, but currently, I get error on app.js because your fork is outdated. Could you write these commands, to fix it: On development branch
`git fetch upstream development` - get the newest changes from the development branch.
`git merge upstream/development` - merge development branch to your own local branch.
`git checkout unique-branch` - visit your current branch
`git merge upstream/development `- merge development branch changes with your current changes. if you using visual studio you and there are conflicts you will have to deal with it manually. After that the usual

```
git status
git add
git commit -m 'message here'
git push
git push orgin [branch name]
```

**Pull request closed without merging**